### PR TITLE
Divya week 2 assignment

### PR DESCRIPTION
**Pull request by 890773**
Week2 assignment- Sorting, Interface, Abstraction, Sum.

### CODE REVIEW

File: Week2/Interface example/InterfaceExample.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+public class InterfaceExample {
+
+    public static void main(String[] args) {
+        
+        Bike bike= new RoyalEnfield();
+        bike.Company();
+        bike.SpeedLimit();
+        bike = new Honda();
```
Comment (VBhaskarachary) [inline Week2/Interface example/InterfaceExample.java]: Please give input as speed and display message as per requirement

File: Week2/Sorting/SortAndSearch.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+public class SortAndSearch {
+    static List<Number> numbers = new ArrayList<>();
+    static Set<Number> noDuplicate = new HashSet<>();
+
+
+
+    public List<Number> removeDuplicates(List<Number> numbers) {
+        List<Number> originalList = new ArrayList<>();
+        noDuplicate.addAll(numbers);
+        originalList.addAll(noDuplicate);
```
Comment (VBhaskarachary) [inline Week2/Sorting/SortAndSearch.java]: Please correct return type for removing unnecessary code

File: Week2/Interface example/InterfaceExample.java (hunk 1 above)
Comment (890773) [reply Week2/Interface example/InterfaceExample.java]: Fixed.

File: Week2/Sorting/SortAndSearch.java (hunk 2 above)
Comment (890773) [reply Week2/Sorting/SortAndSearch.java]: Fixed.

**Review by VBhaskarachary — APPROVED** [review]
Changes looks good

**Pull request merged**